libbeat/docs/command-reference.asciidoc

```diff
@@ -113,6 +113,30 @@ template.
 Exports the current configuration to stdout. If you use the `-c` flag, this
 command exports the configuration that's defined in the specified file.
 
+
+
+*`dashboard`*::
+Exporting a dashboard allows to store a dashboard on disk in a
+module and load it automatically. The following command can be used:
+
+[source,shell]
```

Comment:
Since the example contains an asciidoc reference, you need to replace line 122 with:
```
["source","yaml",subs="attributes"]
```